keyboards/kyria/kyria.h

```diff
@@ -63,3 +63,20 @@
     { R32,   R33,   R34,   R35,   R36,   R37,   R38,   R39   }, \
     { R45,   R46,   R47,   R48,   R49,   KC_NO, KC_NO, KC_NO }, \
 }
+
+#define LAYOUT_2x2u( \
```

Comment:
Actually:
```suggestion
#define LAYOUT_split_3x6_5( \
```

This would be in line with the community layouts for other splits.

Reply:
Thats fair